criu/include/timer.h

```diff
@@ -0,0 +1,12 @@
+#ifndef __CR_TIMER_H__
+#define __CR_TIMER_H__
+
+#include "restorer.h"
+
+extern int prepare_itimers(int pid, struct task_restore_args *args, CoreEntry *core);
```

Comment:
We need to include a header that declares CoreEntry.

Reply:
thanks for your tip, my timer.h is not very well written